### PR TITLE
add bio and avatar for Grant

### PR DESCRIPTION
**Pull request by grant-guo**

### CODE REVIEW

File: _data/authors.yml [hunk 1]
```diff
@@ -169,4 +169,9 @@ vitalii_slobodianyk:
   avatar: vitalii_slobodianyk.svg
   github: slovit
   is_alumni: true
-  
+grant_guo:
+  name: Grant Guo
+  position: Senior Software Developer
+  info: passionate back-end developer interested in applying distributed data computation software to domains, keep curiosity on new things
```
Comment (andricDu) [inline _data/authors.yml]: Capitalize `Passionate`.

Also for grammatical purposes, you might want to reword:
`... , keep curiosity on new things` to something like `..., forever curious about new things.`

**Review by andricDu — CHANGES_REQUESTED** [review]

**Comment by KevinFHartmann** [comment]
"..., forever curious about news things."
That should say "forever curious about new things", unless, of course, you're curious about things on the news. [😊]

________________________________

File: _data/authors.yml (hunk 1 above)
Comment (grant-guo) [reply _data/authors.yml]: Thanks

File: _data/authors.yml [hunk 2]
```diff
@@ -169,4 +169,9 @@ vitalii_slobodianyk:
   avatar: vitalii_slobodianyk.svg
   github: slovit
   is_alumni: true
-  
+grant_guo:
+  name: Grant Guo
+  position: Senior Software Developer
+  info: Passionate back-end developer interested in applying distributed data computation software to domains, forever curious about new things
```
Comment (andricDu) [inline _data/authors.yml]: Missing period at the end, and then you will be good to go.

**Review by andricDu — APPROVED** [review]

**Pull request merged**